README.md

```diff
@@ -7,17 +7,34 @@ the RV32IMC instruction set architecture.
 
 <p align="center"><img src="doc/images/blockdiagram.svg" width="650"></p>
 
-Ibex offers several configuration parameters to meet the needs of various application scenarios.
-The options include two different choices for the architecture of the multiplier and divider unit,
-as well as the possibility to drop the support for the "M" extension completely. In addition, the
-"E" extension can be enabled when opting for a minimum-area configuration.
-
 This core was initially developed as part of the [PULP platform](https://www.pulp-platform.org)
 under the name "Zero-riscy" \[[1](https://doi.org/10.1109/PATMOS.2017.8106976)\], and has been
 contributed to [lowRISC](https://www.lowrisc.org) who maintains it and develops it further. It is
 under active development, with further code cleanups, feature additions, and test and verification
 planned for the future.
 
+## Configuration
+
+Ibex offers several configuration parameters to meet the needs of various application scenarios.
```

Comment:
We should make it clear there are many possible configs it's just the three listed ones are 'supported' (which we also need to define!)